floodviz/views.py

```diff
@@ -34,6 +34,10 @@ def timeseries_data():
     # Hydrodata data clean and write
     j = hydrograph_utils.req_hydrodata(sites, hydro_start_date, hydro_end_date, url_nwis_prefix)
     timeseries_data = hydrograph_utils.parse_hydrodata(j)
+
+    # Write data to file for post build thumbnail
+    with open('floodviz/static/data/hydrograph_data.json', 'w') as f:
```

Comment:
This should not be written to the static directory because it is not data used by the web site. For now let's write it to the new floodviz/thumbnail directory.